csrankings.csv

```diff
@@ -1920,6 +1920,7 @@ Bingkai Lin,Nanjing University,https://sites.google.com/site/bingkai314159,7jXU6
 Bingpeng Ma,Chinese Academy of Sciences,http://people.ucas.edu.cn/~bpma,fClDLgMAAAAJ
 Bingquan Liu,Harbin Institute of Technology,http://homepage.hit.edu.cn/liubingquan,NOSCHOLARPAGE
 Bingsheng He,National University of Singapore,http://www.comp.nus.edu.sg/~hebs,RogYLKYAAAAJ
+Bingsheng Zhang,Zhejiang University,https://www.lancaster.ac.uk/staff/zhangb2/,QnOeC4IAAAAJ

```

Comment:
Can you point to a page with this faculty member's Zhejiang affiliation?

Reply:
Sorry for the late reply. This is the page address: https://person.zju.edu.cn/bingsheng

Reply:
Please update the home page in the PR and I will merge.